hera_sim/tests/test_defaults.py

```diff
@@ -0,0 +1,42 @@
+from os.path import join
+import numpy as np
+from hera_sim.defaults import defaults
+from hera_sim.config import CONFIG_PATH
+from hera_sim.sigchain import gen_bandpass
+from hera_sim.noise import bm_poly_to_omega_p
+
+if defaults._version_is_compatible:
+    def test_config_swap():
+        defaults.set_defaults('h1c')
+        config1 = defaults._config
+        defaults.set_defaults('h2c')
+        assert config1 != defaults._config
+
+    def test_direct_config_path():
+        config = join(CONFIG_PATH, 'HERA_H2C_CONFIG.yaml')
+        defaults.set_defaults(config)
+
+    def test_beam_poly_changes():
+        defaults.set_defaults('h1c')
+        defaults.activate_defaults()
```

Comment:
I would argue that merely calling `defaults.set('h1c')` here should be enough to also `activate` them. If the user is explicitly setting them, they expect them to be active. There should still be the option to deactivate and activate manually, though.

Reply:
I think that's sensible.